src/main/java/dev/goral/rpgmanager/rpgSystems/cpRed/characters/Equipments/CpRedEquipmentsRepository.java

```diff
@@ -0,0 +1,7 @@
+package dev.goral.rpgmanager.rpgSystems.cpRed.characters.Equipments;
+
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CpRedEquipmentsRepository {
```

Comment:
```suggestion
public interface CpRedEquipmentsRepository extends JpaRepository<CpRedEquipments, Long> {
```

tutaj zapomniałeś o rozszerzeniu interfejsu